README.md

```diff
@@ -58,13 +60,44 @@ class TestModel extends Model
 {
     use HasBinaryUuid;
 
-    public function getKeyName()
-    {
-        return 'uuid';
-    }
+    public $incrementing = false;
+    
+    public $primaryKey = 'uuid';
 }
 ```
 
+If you try converting your model to JSON with binary attributes, you will see errors. By declaring your binary attributes in `$uuidAttributes` on your model, you will tell the package to cast those UUID's to text whenever they are converted to array. Also, this adds a dynamic accessor for each of the uuid attributes.
+
+```php
+use Illuminate\Database\Eloquent\Model;
+use Spatie\BinaryUuid\HasBinaryUuid;
+
+class TestModel extends Model
+{
+    use HasBinaryUuid;
+    
+    /**
+     * The suffix for the uuid text attribute 
+     * by default its '_text'
+     * 
+     * @var
+     */
+    protected $uuidTextAttribSuffix = '_str';
```

Comment:
This can be a more clear name. Possibly `uuidSuffix` ? Also see further down for more feedback on this name.

Reply:
`uuidSuffix` is not clear to me. `uuidTextAttribSuffix`  sounds clumsy. I would suggest `textUuidSuffix` or `textualUuidSuffix`.

Additionally, would this belong to configuration or model? Recently in a PR for laravel-status, @freekmurze made me use a configuration option for attribute names. You may want to look at your other packages to get something consistent in the whole Spatie ecosystem :)

Reply:
I was actually stuck between trying to make the property name meaningful while keeping it shortest. You're totally right. This is better